foreign_cc/private/framework/toolchains/linux_commands.bzl

```diff
@@ -38,6 +38,13 @@ def enable_tracing():
 def disable_tracing():
     return "set +x"
 
+def enable_parallel_build():
+    return """
+    _cpu_count=$(nproc)
+    export MAKEFLAGS="-j$_cpu_count${MAKEFLAGS:+ }${MAKEFLAGS:-}"
```

Comment:
I'd also suggest setting `-l$_cpu_count` here just to be a good citizen to the machine as bazel has only assigned a single core to the action in its scheduling and so limiting the parallelism under load is a useful thing to do.
I'm not familiar enough with CMake's parallelism config to know if there's something equivalent there?